Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -3597,7 +3598,7 @@ bool FrameLoader::dispatchBeforeUnloadEvent(Chrome& chrome, FrameLoader* frameLo
 
     {
         SetForScope change(m_pageDismissalEventBeingDispatched, PageDismissalType::BeforeUnload);
-        ForbidPromptsScope forbidPrompts(CheckedPtr { m_frame->page() }.get());
+        ForbidPromptsScope forbidPrompts(m_frame->page());
```

Comment:
Why this revert? Note that this could be written more simply like so:
```
ForbidPromptsScope forbidPrompts(m_frame->checkedPage().get());
```

However, the proposed patch revert security hardening and this should not be done without good reason.

Reply:
That fixed the build error. Thanks!

Reply:
I already landed this patch.